File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex9.java
//Instanciando Scanner
import java.util.Scanner;

public class Ex9{
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	System.out.println("Crédito Especial\n\n");
	
	//vars
	double saldo, credito;
	
	//Entrada
	System.out.print("Insira o saldo médio do cliente: ");
	saldo = ent.nextDouble();
	
	//Processamento
	if ( saldo <= 200 )
		credito = saldo * 0.1;
	else if ( saldo <= 300 )
		credito = saldo * 0.2;
	else if ( saldo <= 400 )
		credito = saldo * 0.25;
	else
		credito = saldo * 0.3;
	
	//Saída
	System.out.printf("Saldo: R$ %.2f\nCrédito: R$%.2f", saldo, credito);
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex9.c
#include <stdio.h>

main()
{
    printf("\n\n");

    //vars
    float saldoMedio, credito;

    //Entrada
    printf("Insira o saldo medio do cliente: ");
    scanf("%f",&saldoMedio);

    //Processamento
    if ( saldoMedio <= 200 )
        credito = saldoMedio * 0.1;
    else if ( saldoMedio <= 300 )
        credito = saldoMedio * 0.2;
    else if ( saldoMedio <= 400 )
        credito = saldoMedio * 0.25;
    else
        credito = saldoMedio * 0.3;

    printf("Saldo medio = R$%.2f\n", saldoMedio);
    printf("Credito = R$%.2f", credito);

}
